README.md

```diff
@@ -1,2 +1,18 @@
-# devfile-sample-java-springboot-basic
-A basic sample application using Java Spring Boot with devfile
+# Creating an application with a Spring Boot code sample
+
+**Note:** The Spring Boot code sample uses the **8081** HTTP port.
+
+Before you begin creating an application with a {ProductName} code sample, it's helpful to understand the relationship between the `devfile` and `Dockerfile` and how they contribute to your build. You can find these files at the following URLs:
```

Comment:
We don't have substitution here for Product Name. It will work on app studio repositories but not here. Just mention "Before using this devfile sample ..." IMO should be good enough